### PR TITLE
Phase1 Final PR

### PR DESCRIPTION
**Pull request by Anand8317**
- Get the list of movie shows from the API call
- Display the movie shows in a grid way.
- Mockup ability to like and comment
- The content is responsive for desktop, tablet, and mobile devices.
- User can post a like by clicking on the like button and it gets stored in the involvement API.
- Website fetches the count of likes from the involvement API and displays it next to the like icon for each item
- Users can click on the category and see the related shows. Next to each category, there is the count for each
- Website design is changed completely to make it more friendly and enhance user experience.
- Added counter function to get all comments and display on the comment popup when clicked
- Added a function to display all comments related to a particular movie on the popup
- Added tests to ensure the counter is working as expected.
- Improved the modal popup look by improving the CSS elements
- Configured babel with the jest testing framework to make it easier to run test files.

### CODE REVIEW

File: src/modules/apiCall.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const dataJson = [''];
+const apiCall = async (homePageOption) => {
+  await fetch(`https://api.tvmaze.com/search/shows?q=${homePageOption[0]}`)
+    .then((data) => data.json())
+    .then((data) => {
+      dataJson.length = 0;
```
Comment (MeqdamAlqudah) [inline src/modules/apiCall.js:6]: - [ ] Please make sure to fix this error and make sure that you really know how the API response data looks like, here it looks like it is a string 

image

File: src/modules/apiCall.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      dataJson.push(data);
+    });
```
Comment (MeqdamAlqudah) [inline src/modules/apiCall.js:8]: - [ ] Please make sure that the comment popup is responsive look how it looks like when the user changes the screen size to mobile, I think you should use more responsive measures. 

| Yours  | original design |
| ------------- | ------------- |
| image  | image |

Comment (MeqdamAlqudah) [inline src/modules/apiCall.js:8]: - [ Optional] Please add validation to the comment section so the user doesn't add a very long comment or name 
image

File: src/modules/apiCall.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  for (let index = 0; index < dataJson[0].length; index += 1) {
```
Comment (MeqdamAlqudah) [inline src/modules/apiCall.js:10]: - [ ] Please make sure that all tests are checked. 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Anand8317 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: Readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [Optional]  improving the colors in the design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/apiCall.js (hunk 1 above)
Comment (Anand8317) [reply src/modules/apiCall.js:6]: The error is now solved.

File: src/modules/apiCall.js (hunk 2 above)
Comment (Anand8317) [reply src/modules/apiCall.js:8]: The comment popup is now responsive

File: src/modules/apiCall.js (hunk 3 above)
Comment (Anand8317) [reply src/modules/apiCall.js:10]: All the tests are passing now.

Comment (Anand8317) [reply src/modules/apiCall.js:10]: ss

File: src/modules/apiCall.js (hunk 2 above)
Comment (OyePriscilla) [reply src/modules/apiCall.js:8]: - [optional] Please I noticed that when a comment is submitted, I need to refresh the browser before the comment can be updated. Kindly work on this in order to enhance the user accessibility of your app. 👍

**Review by OyePriscilla — APPROVED** [review]
Hi @Anand8317 and @admirerbrown 👍 , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

GloriousTenderHalcyon-max-1mb

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**